conf/application.conf

```diff
@@ -136,6 +140,14 @@ microservice {
 mongodb {
     uri = "mongodb://localhost:27017/"${appName}
 
+    pension-scheme-event-reporting-cache {
+            maxSize = 512000
```

Comment:
Why do we have this max-size and this parent naming (pension-scheme-event-reporting-cache) ?